app/controllers/vacancies_controller.rb

```diff
@@ -73,4 +73,8 @@ def sort_column
   def sort_order
     params[:sort_order]
   end
+
+  def searched?
+    params[:commit]
```

Comment:
Worried that this could return true by accident in future if another form is added to the page. I can't foresee what form that might be at the moment but the trouble it would present in the form of a silent bug that would skip past our tests isn't worth a small tweak now:

2 ideas:
- assert that the `commit` value is for our search form eg `params[:commit] && params[:commit].eql?(I18n.t('buttons.apply_filters'))`
- add a new hidden field to the form explicitly for this purpose